lib/bitcoin/BitcoinClient.ts

```diff
@@ -418,6 +419,21 @@ export default class BitcoinClient {
     return BitcoinClient.convertBtcToSatoshis(feerateInBtc);
   }
 
+  /** Get the current estimated fee from RPC or return the stored estimate */
+  private async getEstimatedFeeInSatoshisPerKB (): Promise<number> {

```

Comment:
Style: this method doesn't just "get", it also "updates" stored estimate as a side effect, so I think we should flip the method name, and make the "get" as a side effect if you really want to keep it as a return value.
```suggestion
  private async updateEstimatedFeeInSatoshisPerKB () {
```
or
```suggestion
  private async updateEstimatedFeeInSatoshisPerKB (): Promise<number> {
```

Alternatively you don't have to change the method name, but don't update the cached estimate, update it after this function call.